velox/connectors/hive/HiveConfig.h

```diff
@@ -175,6 +175,9 @@ class HiveConfig {
   static constexpr const char* kSortWriterMaxOutputBytesSession =
       "sort_writer_max_output_bytes";
 
+  static constexpr const char* kS3UseProxyEnvConfig =
+      "hive.s3.use-proxy-env-config";
```

Comment:
How does `hive.s3.use-proxy-from-env` sound?

Reply:
That's good as well :)